### PR TITLE
Documentation for mutators and usage

### LINKED ISSUE #1: Add usage instructions to README
I cloned this repo and tried to run it. Didn't work.
When is the next release? @richardwerkman

### LINKED ISSUE #13: Document AddMutator

### PR DESCRIPTION
**Pull request by richardwerkman**
This pull request adds documentation for all available mutators at the time and how to use the framework.
closes #1 closes #13

### CODE REVIEW

**Comment by nicojs** [comment]
Looks good. What's wrong with the AppVeyor build?

**Comment by richardwerkman** [comment]
I didn't have much time yet to look into it. But it seems to have to do with environment settings on the build server. Things like path seperators and newlines have gotten into some string literals. There will be a seperate pull request for fixing the build.

**Comment by simondel** [comment]
I'd also like to take a look at this. I'll read it tomorrow morning 👍

File: README.md [hunk 1]
```diff
@@ -2,20 +2,28 @@
...
+For an introduction to mutation testing and Stryker's features, see [stryker-mutator.io](https://stryker-mutator.io/).
 
 ## Note
-This project is still in its early days and is not yet available on NuGet
+This project is still in its early days and is not yet available on NuGet. In the meantime, start by [mutation testing your JavaScript](https://stryker-mutator.github.io).
+
+## Getting started
+Stryker.NET offers you mutation testing for your .NET Core projects. It allows you to test your tests by temporarily inserting bugs.
 
 ## Contributing
```
Comment (simondel) [inline README.md]: I would place contributing lower

File: README.md [hunk 2]
```diff
@@ -2,20 +2,28 @@
...
 Want to help develop Stryker.NET? Check out our [contribution guide](/CONTRIBUTING.md).
-In the meantime, start by [mutation testing your JavaScript](https://stryker-mutator.github.io)
 
 ## Compatibility
 Only compatible with .NET Core version 1.1+
 
+## Usage
```
Comment (simondel) [inline README.md]: The info in this readme is not grouped together. I would expect the usage to be described in/near `Getting Started`

File: README.md [hunk 3]
```diff
@@ -2,20 +2,28 @@
...
+For the full documentation about the `dotnet stryker` command, see the [Stryker.CLI readme](/src/Stryker.CLI/README.md).
```
Comment (simondel) [inline README.md]: If you first read this, you have no idea what `dotnet stryker` is and how to get it. Would it be possible to introduce this command a bit more and then metion that the stryker.cli readme describes how to set up your project?

File: src/Stryker.CLI/README.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
+# Stryker.NET Command Line Interface
+The Stryker CLI is currently the only implemented way to use Stryker.NET. 
+
+## Getting started
+To install Stryker.NET on your *test project* add the following lines to your `.csproj` file.
+
+``` XML
+<ItemGroup>
```
Comment (simondel) [inline src/Stryker.CLI/README.md]: Should this be placed at the root or in a specific element?

File: src/Stryker.CLI/README.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+    <PackageReference Include="dotnet-stryker" Version="0.1.*" />
+</ItemGroup>
+```
+
+After adding the references, install the packages by executing `dotnet restore` inside the project folder.
+
+## Usage
+To kick off stryker, execute the following command:
+
+`dotnet stryker`
```
Comment (simondel) [inline src/Stryker.CLI/README.md]: Where should you execute this command? In the root of your repo? In the test project folder?

File: src/Stryker.CLI/README.md [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+
+## Configuration
+While Stryker.NET wants to be a non configuration needed tool, some settings are configurable.
+
+#### Specify your project to mutate
+When Stryker finds two or more project references inside your test project, it needs to know what project should be mutated. Pass the name of this project using:
+
+`dotnet stryker --project SomeProjectName.csproj`
```
Comment (simondel) [inline src/Stryker.CLI/README.md]: If this command is executed in the test project, I would expect the project to be `..\SomeProjectName.csproj`

File: src/Stryker.Core/README.md [hunk 7]
```diff
@@ -0,0 +1,29 @@
+# Mutators
```
Comment (simondel) [inline src/Stryker.Core/README.md]: Where is the AddMutator?

**Pull request merged**